pdm/installers/_editable_install.py

```diff
@@ -2,9 +2,10 @@
 import os
 import sys
 import tokenize
+from pathlib import Path
 
 
-def install(setup_py, prefix, lib_dir, bin_dir):
+def install(setup_py: Path, prefix: Path, lib_dir: Path, bin_dir: Path) -> None:
```

Comment:
```suggestion
def install(setup_py: str, prefix: str, lib_dir: str, bin_dir: str) -> None:
```
These args are taken from `sys.argv`, apparently they are strings